pkg/operator/configmap/configmap_controller.go

```diff
@@ -60,10 +59,12 @@ func NewReconciler(mgr manager.Manager) *ReconcileConfigMap {
 		// Controller only watches this one single configmap.
 		"configmap": fmt.Sprintf("%s/%s", ccoNamespace, configMapName),
 	}
+	context, cancel := context.WithCancel(context.TODO())
```

Comment:
canceling this context doesn't do anything to the context of the leaderelection code. how this this signalling to the leaderelection code to stop running and go on with release on cancel code path?